### DISCUSSION TITLE
After running lua script, the focus is transferred to the output window of the result.

### DISCUSSION THREAD

**Discussion by 9Demien9** [orbitalquark/textadept#258] [General]
After running lua script, the focus is transferred to the output window of the result. 
Of course, you can move back Ctrl+ Alt+N, but is it possible to make the focus itself not move, but remain in the development window?

**Comment by orbitalquark** [comment]
Have you tried setting `textadept.run.run_in_background = true` in your *~/.textadept/init.lua*? https://orbitalquark.github.io/textadept/api.html#textadept.run.run_in_background

**Comment by 9Demien9** [comment]
Thank you very much. It is necessary to read the documentation more carefully)
